carma_wm_ctrl/launch/carma_wm_broadcaster.launch

```diff
@@ -15,6 +15,7 @@
   <arg name = "max_lane_width"  default = "4" doc= "Max lane width in meters within which geofence points are associated to a lanelet as those points are guaranteed to apply to a single lane"/>
   <node name="carma_wm_broadcaster" pkg="carma_wm_ctrl" type="carma_wm_ctrl_node">
     <remap from="georeference" to="$(optenv CARMA_LOCZ_NS)/map_param_loader/georeference"/>
+    <remap from="$(optenv CARMA_ENV_NS)/geofence" to="$(optenv CARMA_MSG_NS)/incoming_geofence_control"/>
```

Comment:
If it works this is fine, but my guess is this remapping will fail because remaps are a direct string match so the from=  part needs to match what is in the source code.